packages/patternfly-4/react-integration/cypress/integration/alert.spec.ts

```diff
@@ -0,0 +1,23 @@
+describe('Alert Demo Test', () => {
```

Comment:
Can you please add some test to verify the action buttons as well.

Reply:
https://github.com/patternfly/patternfly-react/issues/2026  Will add this here.